backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/client/spi/ElasticsearchResponse.java

```diff
@@ -4,27 +4,24 @@
  */
 package org.hibernate.search.backend.elasticsearch.client.spi;
 
-import com.google.gson.JsonObject;
+import java.util.Objects;
 
-import org.apache.http.HttpHost;
+import com.google.gson.JsonObject;
 
 public final class ElasticsearchResponse {
-
-	private final HttpHost host;
+	private final String host;
 	private final int statusCode;
-
 	private final String statusMessage;
-
 	private final JsonObject body;
 
-	public ElasticsearchResponse(HttpHost host, int statusCode, String statusMessage, JsonObject body) {
+	public ElasticsearchResponse(String host, int statusCode, String statusMessage, JsonObject body) {
```

Comment:
This one is pretty straightforward, we only use the host to log the response, and it's an SPI ... so removing the apache class here should be "safe"